.clang-format

```diff
@@ -1,27 +1,19 @@
 # @file      .clang-format
 # @brief     clang-format configuration
 #
-# MIT License
+# Copyright joelguittet and mender-mcu-client contributors
```

Comment:
The `Copyright` lines can have a year specifying the last year the copyright was updated (when the code was changed by the copyright owner/holder). But it's not strictly necessary, there are other means to determine that.

Reply:
I know that and because it's not mandatory I think, I have just removed the year. The format of the notice on each file is verified by the ci pipeline, so it s easier to check without dealing with checking if the date is valid.